benchto-driver/src/main/java/io/trino/benchto/driver/BenchmarkProperties.java

```diff
@@ -79,6 +79,9 @@
     @Value("${macroExecutions.healthCheck:#{null}}")
     private String healthCheckMacros;
 
+    @Value("${repeatLevel:QUERY}")
```

Comment:
WDYT about deprecating `runs` and introducing `query-runs` and `benchmark-runs`?

Reply:
As followup yes. We need rapidly stable results of benchmarks.

Reply:
So I'd suggest only adding `query-runs` (or `repeat-queries`) now, instead of the `repeatLevel` enum.

Reply:
How to enable repeating on benchmark / query level using only `query-runs`?

Reply:
`runs` would set the number of repeats for benchmarks and `query-runs` would set the number of repeats for every query.

Reply:
ok, it is not backward compatible change (by default). We expect that by default the queries will be repeated instead of benchmarks. If not backward compatible for default case change is ok, I can do it. Let's wait for other reviewers.

Reply:
My main concern is that this is a public API and I'd avoid breaking changes as much as possible. Adding new options knowing they'll get deprecated in near future doesn't sound right.